package.json

```diff
@@ -13,7 +13,8 @@
 		"lint:fix": "eslint . --ext ts,.tsx --fix",
 		"test:unit": "jest",
 		"test:e2e": "start-server-and-test 'npm run build && npm run start' http-get://localhost:3000 'synpress run'",
-		"test:e2e:ci": "synpress run --group smoke-tests --record --parallel",
```

Comment:
We need to keep `test:e2e:ci` because we will want to run these tests in CI eventually for automated testing

Reply:
Okay will add this back.